app/metrics.yaml

```diff
@@ -540,6 +540,29 @@ events:
     metadata:
       tags:
         - Sharing
+  share_to_app:
+    type: event
+    description: |
+      A user selected an app to share to from the share sheet.
+    extra_keys:
+      app_package:
+        description: |
+          Package identifier for app or action the user selected
+          (ex. 'org.mozilla.focus' or 'com.android.bluetooth')
+        type: string
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/28033
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/28118#issuecomment-1341620815
+    notification_emails:
+      - android-probes@mozilla.com
+    data_sensitivity:
+      - interaction
+      - technical
```

Comment:
As stated in your data review request, this should just be interaction. Please remove `-technical`